src/Jackett.Common/Indexers/NCore.cs

```diff
@@ -208,9 +208,25 @@ public NCore(IIndexerConfigurationService configService, WebClient wc, Logger l,
         
                     else
                     {
-                        Match m = Regex.Match(release.Title, @""+ seasonep + @"\s?$", RegexOptions.IgnoreCase);
                         if (query.MatchQueryStringAND(release.Title, null, seasonep))
                         {
+                            /* For sonnar if the search querry was english the title must be english also so we need to change the Description and Title */
+                            var temp = release.Title;
+                            
+                            // releasedata everithing after Name.S0Xe0X
+                            var releasedata =release.Title.Split(new[] { seasonep }, StringSplitOptions.None)[1].Trim();
+                            
+                            /* if the release name not contains the language we add it because it is know from category */
+                            if (cat.Contains("hun") && !releasedata.Contains("hun"))
+                                releasedata += ".hun";
+
+                            // release description contains [imdb: 8.7] but we only need the data before it for title
+                            var description = release.Description.Split('[');
+                            
+                            release.Title = (description[0].Trim() + "." + seasonep.Trim() + "."+ releasedata.Trim('.')).Replace(' ', '.');
+
+                            // add back imdb points to the description [imdb: 8.7]
+                            release.Description = temp+" ["+ description[1];
```

Comment:
If there's no [imdb: ] tag (series which are not available on imdb?) this would cause an exception too. i suggest you check the length of description before this.

Reply:
I handle this with the new commit. 
Also correct that if we search for S0X the tiles were S0X,E0X